ogr/abstract.py

```diff
@@ -117,7 +117,8 @@ def __str__(self) -> str:
 
 
 class IssueStatus(IntEnum):
-    open = 1
+    opened = 1
+    open = opened
```

Comment:
Another reason, why `open` is not well named is that it shadows the built-in `open` function.